content/en/blog/_posts/2025-12-17-kubernetes-v1-35-release/index.md

```diff
@@ -0,0 +1,476 @@
+
+Kubernetes v1.35 introduces native Gang Scheduling support via the new Workload API and PodGroup concept. This feature implements an "all-or-nothing" scheduling strategy, ensuring that a defined group of Pods is scheduled only if the cluster has sufficient resources to accommodate the entire group simultaneously.
+
+The primary benefit is improved reliability and efficiency for batch and parallel workloads. By preventing partial deployments, it eliminates resource deadlocks and ensures that expensive cluster capacity is utilized only when a complete job can run, significantly optimizing the orchestration of large-scale data processing tasks.
+
+This work was done as part of [KEP #4671](https://kep.k8s.io/4671) led by SIG Scheduling.
+
+### Extended toleration operators for threshold-based placement
+
+Kubernetes v1.35 introduces SLA-aware scheduling by enabling workloads to express reliability requirements. The feature adds numeric comparison operators to tolerations, allowing pods to match or avoid nodes based on SLA-oriented taints such as service guarantees or fault-domain quality.
+
+The primary benefit is enhancing the scheduler with more precise placement. As critical workloads can demand higher-SLA nodes, while lower priority workloads can opt into lower SLA ones. This improves utilization and reduce cost without compromising reliability.
+
+This work was done as part of [KEP #5471](https://kep.k8s.io/5471) led by SIG Scheduling.
+
+### Deployment status: count of terminating replicas
+
+Historically, the Deployment status provided details on available and updated replicas but lacked explicit visibility into Pods that were in the process of shutting down. A drawback of this omission was that users and controllers could not easily distinguish between a stable Deployment and one that still had Pods executing cleanup tasks or adhering to long grace periods.
+
+Kubernetes v1.35 promotes the `terminatingReplicas` field within the Deployment status to Beta. This field provides a count of Pods that have a deletion timestamp set but have not yet been removed from the system. This feature is a foundational step in a larger initiative to improve how Deployments handle Pod replacement, laying the groundwork for future policies regarding when to create new Pods during a rollout.
+
+The primary benefit is improved observability for lifecycle management tools and operators. By exposing the number of terminating Pods, external systems can now make more informed decisions such as waiting for a complete shutdown before proceeding with subsequent tasks without needing to manually query and filter individual Pod lists.
+
+This work was done as part of [KEP #3973](https://kep.k8s.io/3973) led by SIG Apps.
```

Comment:
This looks great now, thank you! But I believe it should move to beta section, right?